src/views/PostCreator/components/HeaderPreview/HeaderPreview.css.js

```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components';
+
+export const StyledImage = styled.img``;
```

Comment:
to testowy pewnie ale smieszne w sumie xD

Reply:
Na przyszłość xD